definitions/firefox_desktop.toml

```diff
@@ -1183,7 +1204,18 @@ description = "Normalized Country Code"
 
 
 [data_sources]
-  
+
+[data_sources.active_users_aggregates_v1]
+from_expression = """(
+    SELECT *
+     FROM `moz-fx-data-shared-prod.telemetry.active_users_aggregates`
+    WHERE app_name = 'Firefox Desktop'
+)"""
+friendly_name = "Active Users Aggregates"
+description = "Active Users Aggregates, filtered on Firefox Desktop"
+submission_date_column = "submission_date"
+client_id_column = "NULL"  # this table doesn't include client_id, and we don't need it for calculating DAU
```

Comment:
@scholtzan is `"NULL"` here the correct value?

Reply:
I _think_ this should work, based on Anna's previous comment: https://github.com/mozilla/metric-hub/pull/109#discussion_r1202645805

Reply:
Sorry I thought I edited this comment before sending, but this didn't need to be exclusively a question for Anna. The point was just to confirm that this is the correct format for specifying a null client_id.

Reply:
Ok, sounds good, thanks!